<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pyramidGame</class>
 <widget class="QMainWindow" name="pyramidGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1173</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0">
     <widget class="QLabel" name="pyramid_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
margin:0% 200%;</string>
      </property>
      <property name="text">
       <string>Pyramid Tic Tac Toe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="pyramid">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QFrame" name="boardpyramid_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #cbbf7a;
color:rgb(0,0,0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>10</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="x13_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QFrame" name="boardpyramid_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>614</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #cbbf7a;
color:rgb(0,0,0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>10</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="5">
           <widget class="QPushButton" name="x35_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="x31_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="x32_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(0,0,0);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="x34_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(0,0,0);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="x33_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QFrame" name="boardpyramid_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>382</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #cbbf7a;
color:rgb(0,0,0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>10</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="x23_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(0,0,0);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="x22_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="x24_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(81, 81, 81);
font: 700 22pt &quot;Segoe UI&quot;;
}

QPushButton:hover{
	background-color: #fff;
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="pyramid_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="styleSheet">
          <string>color: white;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QFrame" name="frame_9">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="ScoreX_2">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>5</number>
               </property>
               <property name="text">
                <string>Score (X) </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_10">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="ScoreO_2">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="lineWidth">
                <number>5</number>
               </property>
               <property name="text">
                <string>Score (O)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="helpButton_2">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
background: #cbbf7a;
    color: #000000; 
    border: 2px solid rgb(0,0,0) ;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {

background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
	stop:0 #000,
	stop:0.5 #52a5e9 ,
	stop:1 #000 ); 
	color: white;
}</string>
            </property>
            <property name="text">
             <string>HELP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exitButton_2">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
background: #cbbf7a;
    color: #000000; 
    border: 2px solid rgb(0,0,0) ;
    border-radius: 20px;
    padding: 5px;
}

QPushButton:hover {

background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
	stop:0 #000,
	stop:0.5 rgb(236, 87, 102) ,
	stop:1 #000 ); 
	color: white;
}</string>
            </property>
            <property name="text">
             <string>EXIT</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
